### ISSUE TITLE
DateTime to Timestamp back to DateTime is different

### ISSUE THREAD

**Issue by mRy0** [bridgedotnet/Bridge#1901]
### Problem

If I parse a DateTime to a Timestamp and back to DateTime it is wrong about a hour

### Actual

(DateTime.Now)     Thu Sep 29 2016 20:41:40 GMT+0200 (CEST)
(TimeStamp)          1475181700
(Parsed Back)         Thu Sep 29 2016 21:41:40 GMT+0200 (CEST)

### Steps To Reproduce

http://deck.net/1bd273ce6915a8c39d55cbf0ee7e9a7b

```cs
public class Program
{
    public static void Main()
    {
        var now = DateTime.Now;
        Console.WriteLine(now);

        var unixNow = DatetimeToUnixTime(now);        
        Console.WriteLine(unixNow);

        var parsedUnixNow  = UnixTimeToDatetime(unixNow);        
        Console.WriteLine(parsedUnixNow); 
    }
    
    public static long DatetimeToUnixTime(DateTime time)
    {
        return (long) time.Subtract(new DateTime(1970,1,1)).TotalSeconds;
    }

    public static DateTime UnixTimeToDatetime(long time)
    {
        return new DateTime(1970,1,1).AddSeconds(time);
    }
}
```

**Comment by geoffreymcgill** [comment]
@mRy0, 

Thanks for submitting this defect. We will fix and try to include in the upcoming Bridge 15.2 release. 

Looks like a simple error of not properly comparing two Dates by their UTC equivalents.

**Comment by geoffreymcgill** [comment]
A pull-request (#1908) has been submitted with a fix for this issue. The fix should be included in the upcoming Bridge 15.2.0 release.
